opennms-webapp/src/main/webapp/WEB-INF/jsp/graph/results.jsp

```diff
@@ -198,6 +199,29 @@
             </c:if>
         </h3>
      </div> <!-- panel-heading -->
+       <c:if test="${fn:contains(resultSet.resource.resourceType.label, 'SNMP Interface')}">
+         <c:if test="${fn:contains(resultSet.resource.label,'(*)') != true}">
+           <c:forEach var="attribute" items="${resultSet.resource.attributes}">
+             <c:if test="${fn:contains(attribute.name, 'ifIndex')}">
+               <c:set var="ifIndex" value="${attribute.value}"/>
+             </c:if>
+             <c:if test="${fn:contains(attribute.name, 'nodeId')}">
+               <c:set var="nodeId" value="${attribute.value}"/>
+             </c:if>
+           </c:forEach>
+           <%
+           String grafanaProtocol = System.getProperty("org.opennms.grafanaBox.protocol", "http");
+           String grafanaHostname = System.getProperty("org.opennms.grafanaBox.hostname", "localhost");
+           int grafanaPort = Integer.parseInt(System.getProperty("org.opennms.grafanaBox.port", "3000"));
+
```

Comment:
Can we put these properties in the response from the controller/command instead of manually defining them here in the jsp?